src/routes/Chat/component.js

```diff
@@ -148,6 +149,7 @@ export default class Chat extends Component {
 			>
 				<Screen.Content nopadding>
 					{ incomingCallAlert && !!incomingCallAlert.show && <CallNotification { ...incomingCallAlert } dispatch={dispatch} />}
+					{ incomingCallAlert && incomingCallAlert.callstatus === 'accept' ? <div> <CallIframe { ...incomingCallAlert } /> </div> : null }
```

Comment:
Why surround `CallIFrame` with a new div? I don't  any use of it here. Please correct me if I'm missing something